recipe/meta.yaml

```diff
@@ -16,14 +16,15 @@ source:
     - flang-support.patch
 
 build:
-  number: 4
+  number: 5
   skip: true  # [win and vc<14]
 
 requirements:
   build:
     # dummy unused top-level requirements
     # needed to construct the right conda-smithy matrix
-    - {{ compiler('fortran') }}
+    - {{ compiler('fortran') }}  # [not win]
+    - flang  # [win]
```

Comment:
Note that we already set `fortran_compiler: flang` on windows, so you can leave the compiler-jinja also on windows, and simply set 
```
fortran_compiler_version:  # [win]
  - 18                     # [win]
```
in CBC

Reply:
Done.

Reply:
The specification of `fortran_compiler` is [redundant](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/74c2e63226d566984c6219aaf8b6629ea0b48bf9/recipe/conda_build_config.yaml#L38) with the global pinning, but 👍